### ISSUE TITLE
More waterproof containers/backpacks

### ISSUE THREAD

**Issue by randomtyper** [CleverRaven/Cataclysm-DDA#65879]
### Is your feature request related to a problem? Please describe.

Currently there is little content regarding water exploration, and not much equipment related to it. One of pieces that misses and would be very helpful to the players in case of expansion on such content are large wearable waterproof bags. 

Sure, there are one-gallon zipper bags, but they are pretty small, do not carry much weight and cannot be worn. The in game dive bag, one large container that would be logical to be made waterproof, is actually a non-waterproof variation. There is also a jerrypack, but it is very small and only usable for liquids and very small items.

I think expansion on this matter would be good both for gameplay and realism. It would add better variety in containers (waterproof bags have less pockets and overall storage than land backpacks, but higher utility), while addition of such dry bags will allow to make much more items spoilable/breakable by water, like paper items and food, while not locking players out of transporting those across water obstacles.

### Solution you would like.

Add new items:

1. A 30L dry bag. Basically a waterproof version of the in-game dive bag (a bit heavier and carries more weight, though). Obviously, it can be placed inside another bag to act as waterproof lining to it. I find it somenthing many real survivors would do if they got hands on one. 

Should spawn where wet mesh dive bags spawn (dive shops, freshwater stations, aquatic humanoids).

May be crafted using waterproofing proficiency (duh), some plastic, polycarbonate/plastic sheets, duct tape and cloth/leather. Skill levels tailoring 5 and fabrication 3-ish.

2. A survivor variation with some waterproof pockets added to its sides. Or just open pockets for bottles/small dive tanks. May use a dry bag as one of components instead of plastic and polycarbonate. 6-7 tailoring and 4-5 fabrication seems reasonable to me.

3. A waterproof duffel. 35-60 liter monstrosity that will fit and protect almost everything. Just don't try to fight with it. Most likely uncraftable.

4. A large waterproof bag (60-80L). No shoulder straps, but takes little space in unfilled state and can be used as a large waterproofing liner for something like military backpack. There may be great synergy with pack frame too. May be craftable at the same level as survivor drypack.

5. Backpack/flotation vest hybrid (in terms of game mechanics) with pockets for weights, small scuba tank and a space for either a fullsize scuba tank or special waterproof backpack module. IRL it is a real thing, if a bit of a novelty. Uncraftable or with a survivor version cobbled from a flotation vest, a belt with some pouches and some belts to fix tank/dry bag.

6. Military diver waterproof bags for MOLLE (Maybe). Or a full-on hardcase container like the one used by Green Light teams for their nukes.

Some additional ideas (or make these a separate issue?):
1.It would be nice if player was able to strap a small scuba tank to the chest of the character, or to the belt. That way it would be possible to use one as backup, just like IRL.
2. Maybe increase environmental protection of survivor divemask to 4 to match that of swim goggles and other masks (or even increase to 15 of survivor goggles, but that is too much imo)? After all, there is thick glass between environment and wearer's face, and even if air contacts with it, it has to go through the mask, even if it is inactive.

### Describe alternatives you have considered.

1. Reworking current dive bag into waterproof bag.
2. Implementing the ideas partially.
3. Continue swimming around with sandwiches and books in pockets.

### Additional context

Examples:
1. Example of a 30L dry bag: https://www.amazon.com/KastKing-Waterproof-Military-Construction-Swimming/dp/B08ZXDZKJQ
2. Demonstration of making a handmade 30L dry bag: https://www.youtube.com/watch?v=EkFIMpKuGv0
3. What a survivor drypack is intended to look like: https://www.amazon.com/Waterproof-Backpack-Kayaking-Outdoors-Activities/dp/B0B1Q48F7X?th=1
4. Waterproof duffel example: https://jacksonandcole.com/products/jackson-cole-waterproof-duffel-bag-backpack-large-dry-bag-60l-zip
5. Large dry bag example: http://www.incept.co.nz/dry-bags/db60-dry-bag-60l
6. Backpack/buoyancy compensator combo: https://www.amazon.com/OCEANIC-JETPACK-COMPLETE-DIVING-BACKPACK/dp/B00SBYZ9NA

**Comment by estebandellasilva** [comment]
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/89189872/895a0355-ccc3-4a09-84b2-807015d95673)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/89189872/1b1e1bf1-b326-4918-bda1-975b13fe1b78)
its kinda understandable why the dive bag is not waterproof, if you look at the description

**Comment by fairyarmadillo** [comment]
I guess this is (maybe?) a problem for ziplocs too, but there's an issue with large waterproof bags where the player is now somehow able to insert and remove items from the bag while underwater without getting everything in the bag wet. I don't think a backpack-sized item should be merged until a solution is in place.

The only thing I can think of that doesn't completely suck would be a prompt for when you try to take or remove an item from a waterproof bag while underwater. The other idea would be to make waterproof items activatable to seal or unseal them, but that sounds even more annoying.
